### PR TITLE
Introducing fast / verbose queries

### PR DESCRIPTION
**Pull request by tomgco**

### CODE REVIEW

File: test/invoke.search.hotel.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ var assert = require('assert');
 var index = require('../index');
 
 describe('Integration: search.hotel', function () {
-  it('should find results for amenities and tiles', function (done) {
+  it('hjshould find results for amenities and tiles', function (done) {
```
Comment (nelsonic) [inline test/invoke.search.hotel.js]: hjshould?

Comment (tomgco) [inline test/invoke.search.hotel.js]: My vim foo showing up here, will fix :D

**Comment by jruts** [comment]
@tomgco looks good. Can you add a small line in the README to explain it + update the version in package.json so we can publish?

**Comment by tomgco** [comment]
> @tomgco looks good. Can you add a small line in the README to explain it + update the version in package.json so we can publish?

My pleasure.

**Comment by tomgco** [comment]
Not sure why it is deciding to fail on codeship now, nothing has changed code-wise.

Seems to be flakey :/

**Comment by nelsonic** [comment]
@tomgco suspect that increasing the timeout for the tests will ensure this passes... 
https://codeship.com/projects/145475/builds/15383176
both errors are due to timeout... a [`mocha.opts`](https://github.com/numo-labs/lambda-ne-classic-package-provider/blob/853546ed599395e895efa04c18a0a1226b0c3581/test/mocha.opts) file in the `/test` directory should do the trick.

**Comment by tomgco** [comment]
> both errors are due to timeout... a mocha.opts file in the /test directory should do the trick.

They have been hard-coded in the tests with `this.timeout`. I have just updated them.

**Comment by tomgco** [comment]
We are green @jruts @nelsonic :D :dancer:

**Comment by nelsonic** [comment]
@tomgco nice one! 🚀

**Comment by nelsonic** [comment]
New version on NPM: `taggable-searcher@5.1.0` ✅

**Pull request merged**